src/components/Article.jsx

```diff
@@ -1,25 +1,48 @@
 import { useState } from "react";
-import getArticle from "./GetArticle";
+
 import { useEffect } from "react";
 import { useParams } from "react-router-dom";
+import { getCommentByArticleID, getSingleArticle } from "../api/APICalls";
 
-export default function Article(articleID) {
+export default function Article() {
   const { article_id } = useParams();
-  const bothArticleID = article_id || articleID.value;
   const [article, setArticle] = useState({});
+  const [comments, setComments] = useState([]);
+
   useEffect(() => {
-    getArticle(bothArticleID).then((response) => {
+    getSingleArticle(article_id).then((response) => {
       setArticle(response);
     });
-  }, [bothArticleID]);
+  }, [article_id]);
 
+  useEffect(() => {
+    getCommentByArticleID(article_id).then(
+      (response) => {
+        setComments(response);
+      },
+      [article_id]
+    );
+  });
```

Comment:
This should be along with the comments being rendered below, be extracted out into its own component. This is also missing the dependency array, without any dependency array we are allowing this to always trigger when anything re-renders this component which is not idea. An empty dependency array here would only cause this to trigger on load, which is the behaviour we want.